pwa/src/components/subscribers/subscriberForm.tsx

```diff
@@ -196,138 +218,85 @@ export const SubscriberForm: React.FC<SubscriberFormProps> = ({ subscriberId, en
-                          name="description"
-                          id="descriptionInput"
-                          defaultValue={subscriber?.description}
-                        />
+                        <Textarea name="description" label="Description" {...{ register, errors }} />
                       </div>
                     </div>
                     <div className="row form-row">
                       <div className="col-6">
-                        <SelectInputComponent
-                          options={[
-                            { name: "GET", value: "GET" },
-                            { name: "POST", value: "POST" },
-                            { name: "PUT", value: "PUT" },
-                            { name: "PATCH", value: "PATCH" },
-                          ]}
-                          name={"method"}
-                          id={"methodInput"}
-                          nameOverride={"Method"}
-                          data={subscriber?.method}
-                          required
+                        <SelectSingle
+                          options={methodSelectOptions}
+                          name="method"
+                          label="Method"
+                          {...{ control, errors }}
+                          validation={{ required: true }}
                         />
                       </div>
                       <div className="col-6">
-                        <GenericInputComponent
-                          type={"number"}
-                          name={"runOrder"}
-                          id={"runOrderInput"}
-                          data={subscriber?.runOrder}
-                          nameOverride={"Order"}
-                        />
+                        <InputNumber name="runOrder" label="Run order" {...{ register, errors }} />
                       </div>
                     </div>
                     <div className="row form-row">
                       <div className="col-6">
-                        <TextareaGroup
+                        <Textarea
                           name="conditions"
-                          label={<span>Conditions (JSON Logic) <a target="_blank" href="https://docs.conductor-gateway.app/en/latest/features/handlers/">Read more about the use of JSON Logic</a></span>}
-                          id="conditionsInput"
-                          defaultValue={subscriber?.conditions}
+                          label={"Conditions (JSON Logic)"}
+                          tooltipContent={
+                            <a target="_blank" href="https://docs.conductor-gateway.app/en/latest/features/handlers/">
+                              Read more about the use of JSON Logic
+                            </a>
+                          }
+                          {...{ register, errors }}
+                          validation={{ validate: () => validateJSON(getValues("conditions")) }}
                         />
```

Comment:
This field has unintentionally become a `required` field due to not passing the validate when it's empty.

Reply:
I checked this thread https://github.com/react-hook-form/react-hook-form/issues/1781 for some answers. It seems that the only (and preferred by the creator of this package) just not validating it when there's no data available is the solution. 

I don't understand why the validation kicks in at all when it's empty and not required. That being said, we can easily fix this. 

I've left a comment on the validateJSON function with a fix 👍🏼